guide/webpack.config.js

```diff
@@ -32,6 +33,15 @@ module.exports = validate({
                 include: RegExp(__dirname + "/src"),
                 loaders:["react-hot", "babel-loader"]
             },
+	    {
+		test: /\.txt$/,
+		loader: 'raw-loader',
+		include: path.resolve(__dirname, 'src/examples/raw-code'),
+	    },
+	    {
+		test: /\.md$/,
+		loader: 'raw-loader',
+	    },
```

Comment:
This loader is an _awesome_ way for the component library to avoid DRY violations. That's great! As a team, you're using us a great new feature of a loader, but it's not totally clear to me if these `.txt` & `.md` are required. Are they?

I can see from the loader's [README example](https://github.com/webpack-contrib/raw-loader/tree/master#inline), the loader is _generally_ invoked with:
```javascript
import txt from 'raw-loader!./file.txt';
```

In the pull request, we're using this syntax:

```javascript
import VerticalMenuExCode from '!raw!./examples/VerticalMenuEx';
```

My hunch is that this syntax is the Webpack 1.* way of including the loader.